lib/mutant/cache.rb

```diff
@@ -1,14 +1,17 @@
 module Mutant
   # An AST cache
   class Cache
-    include Equalizer.new, Adamantium::Mutable
+    include Concord.new(:file), Adamantium::Mutable
 
     # Initialize object
     #
+    # @param [Class<File>]
+    #
     # @return [undefined]
     #
     # @api private
-    def initialize
+    def initialize(*)
+      super
       @cache = {}
```

Comment:
@dkubb No all parsing happens in single threaded bootup code. Mutant loads all code upfront.